source/_posts/2016-01-25-upgrading-to-ruby-on-rails-5-0.markdown

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: "Upgrading to Ruby on Rails 5.0"
+category:
+comments: true
+author:
+  name: Santiago Pastorino
+  email: santiago@wyeworks.com
+  twitter_handle: spastorino
+  github_handle: spastorino
+  image: /images/team/santiago-pastorino.jpg
+  description:
+---
+
+In this post, we’re going to update the Rails 4.2.5 application to Rails 5.0.0.beta1. If we didn’t already have that version, we would have to first update to Rails 4.2.5 and clean up the possible warnings generated. A good test suite is recommended - our application has them - but if we didn’t have them, it would be good to be able to add a few automated tests that lend us some confidence before jumping into the new Rails 5.0. Not doing so would confront us with too many risks. 
+
+What's the first thing one does when updating a Rails application? Personally, the first thing I try to do is become well-informed regarding the changes the new application includes. To do this, I generally look at the Rails upgrade guide - in our case [Upgrading from Rails 4.2 to Rails 5.0](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-2-to-rails-5-0) - and I take a look at the CHANGELOG.md file for each one of the frameworks. In my case, given that I form part of the Rails core team, I follow its development rather closely, which makes it a bit easier for me to stay on top of everything. At any rate, following the upgrade guide and taking a look at the CHANGELOGs should be sufficient, although if you’re interested in seeing what’s currently happening, you can follow the day-to-day changes that are introduced in Rails. 
+
+From the upgrade guide (still very inadequate), we learn that some of the changes in Rails 5.0 are:
```

Comment:
So, the change would be

"From the upgrade guide (still very inadequate), we learn that some of the changes in Rails 5.0 are" =>
"From the upgrade guide (still a bit incomplete), we learn that some of the changes in Rails 5.0 are"